lib/assets/chart_builder/main.css

```diff
@@ -124,6 +124,10 @@ input[type=number] {
   margin-bottom: 0.3em;
 }
 
+.info-box p:first-child {
+  margin-top: 0;
```

Comment:
Ah this kills default browser styles, maybe we should have `margin: 0` on the paragraphs and start from there to make sure we don't rely on the defaults?

Reply:
Done!